packages/ui-kit/src/themes/default.types.ts

```diff
@@ -109,4 +123,13 @@ export type Styles = {
       backgroundColor?: string
     }
   }
+  yAxis?: {
+    beginAtZero?: boolean
+    scale?: 'linear' | 'logarithmic'
+  }
 }
+
+/**
+ * @deprecated the type is deprecated, use ChartStyles instead
```

Comment:
1. [Those types](https://github.com/propeldata/ui-kit/pull/99/files#diff-07efbf26029756cd7c54b04ab90507ea83b403061e65e37558e8214382b6897fR16) are related to Chartjs API. Since we already have `ChartPaddingOptions` and `ChartPlugins` I suggested renaming from common `Styles` to more specific `ChartStyles`.
2. It's all about the shared context. `Leaderboard` had `ChartVariant` type export and `TimeSeries` also had `ChartVariant` type export. It leads to a duplicate name conflict. Actually, I'll restore `ChartVariant` type in one of the components and add the deprecated note. It is always good to leave as many warnings for developers as possible.